packages/driver/test/cypress/integration/commands/assertions_spec.js

```diff
@@ -976,6 +976,57 @@ describe('src/cy/commands/assertions', () => {
     })
   })
 
+  context('format quotation marks', () => {
+    it('preserves quotation marks in number strings', function (done) {
+      cy.on('log:added', (attrs, log) => {
+        if (attrs.name === 'assert') {
+          cy.removeAllListeners('log:added')
+
+          expect(log.get('message')).to.eq('expected **25** to equal **\'25\'**')
+
+          done()
+        }
+      })
+
+      cy.then(() => {
+        try {
+          expect(25).to.eq('25')
+        } catch (error) {} /* eslint-disable-line no-empty */
+      })
+    })
+
+    it('preserves quotation marks in empty string', function () {
+      try {
+        expect(42).to.eq('')
+      } catch (error) {} /* eslint-disable-line no-empty */
+    })
+
+    it('preserves quotation marks if escaped', function () {
+      expect('\'cypress\'').to.eq('\'cypress\'')
```

Comment:
we should use template literals instead of having to escape the single quotes

Reply:
Done.